json.c

```diff
@@ -271,6 +295,38 @@ static struct json_object *mode_id_info(int fd, uint32_t blob_id)
 	return obj;
 }
 
+static struct json_object *hdr_output_metadata_info(int fd, uint32_t blob_id)
+{
+	drmModePropertyBlobRes *blob = drmModeGetPropertyBlob(fd, blob_id);
+	if (!blob) {
+		perror("drmModeGetPropertyBlob");
+		return NULL;
+	}
+
+	struct hdr_output_metadata *meta = blob->data;
+
+	struct json_object *obj = json_object_new_object();
+	json_object_object_add(obj, "type", new_json_object_uint64(meta->metadata_type));
+	const struct hdr_metadata_infoframe* info = &meta->hdmi_metadata_type1;
```

Comment:
We should only do this if the type is set to TYPE1.

Reply:
Right! Done.